lib/serializer/CTypeList.h

```diff
@@ -85,7 +85,7 @@ class DLL_LINKAGE CTypeList: public boost::noncopyable
 
 	/// Returns sequence of types starting from "from" and ending on "to". Every next type is derived from the previous.
 	/// Throws if there is no link registered.
-	std::vector<TypeInfoPtr> castSequence(TypeInfoPtr from, TypeInfoPtr to) const;
+	static std::vector<TypeInfoPtr> castSequence(TypeInfoPtr from, TypeInfoPtr to);
```

Comment:
Again, don't see a reason to declare method static unless method *needs* to be called without isntance of object.
Please 
a) convert this into a static function insider .cpp file
b) revert change (probably better IMO)

Reply:
reverted